packages/flame_svg/test/svg_test.dart

```diff
@@ -93,5 +93,39 @@ void main() {
         );
       },
     );
+
+    testWidgets(
+      'render sharply with viewfinder zoom',
+      (tester) async {
+        addTearDown(() async {
+          await tester.binding.setSurfaceSize(null);
+        });
+
+        final flameSvg = await _parseSvgFromTestFile(
+          'test/_resources/hand.svg',
+        );
+
+        await tester.binding.setSurfaceSize(const Size(100, 100));
+
+        await tester.pumpWidget(
+          MaterialApp(
+            debugShowCheckedModeBanner: false,
+            home: Transform.scale(
+              scale: 2,
+              child: CustomPaint(
+                painter: _SvgPainter(flameSvg),
+              ),
+            ),
+          ),
+        );
+
+        await tester.pumpAndSettle();
+        await expectLater(
+          find.byType(MaterialApp),
+          matchesGoldenFile(
+              './_goldens/render_sharply_with_viewfinder_zoom.png'),
```

Comment:
```suggestion
              './_goldens/render_sharply_with_viewfinder_zoom.png'),
```
Add a trailing comma here to make the analyzer happy.

Reply:
Applied in 5a5a302188c928ba54027f846977f0850b4358c2